tools/run_tests/sanity/check_bazel_workspace.py

```diff
@@ -82,6 +83,7 @@
     'io_bazel_rules_go',
     'build_bazel_rules_apple',
     'build_bazel_apple_support',
+    'com_github_google_re2',
```

Comment:
this seems wrong. re2 is not a "bazel only dep". We include it both from the bazel BUILD and from the third_party/re2 submodule. Remove this line?